### ISSUE TITLE
State of this project vs Wagtail flexible forms?

### ISSUE THREAD

**Issue by johnraz** [AccentDesign/wagtailstreamforms#124]
Hi there,

I'm currently working on a project that requires some smart handling of forms to gather various visitor feedback.
Those forms often have to change depending on the content they will be linked to.

Anyway, this is how I ended up here but I'm now struggling to decide if I should use https://github.com/noripyt/wagtail-flexible-forms which seems way more "alpha-ish" than this project but on the other hand, seems to be considered the "official" way.

Could you give me your feeling about this?
Thanks!

**Comment by stuartaccent** [comment]
Hi @johnraz 

Hope you are well, Im not in a position to comment on that package, this is the first I've heard of it.

This project was built purely for our own purposes because the internal wagtail forms simply weren't enough for our needs and we simply made it public. Its use is only really for a couple of our clients. There is no planned development for this currently, however we are keeping it workable with current wagtail versions to help the community. At present I am the only one who actively maintains this package with the exception of the odd few. I know looking through wagtails issue they did plan at one point to do more with forms but ive not heard anything in months maybe that package is the start of it or a concept (I don't know).

Ive tried to create it in a way that can be used for multiple purposes that wouldn't require much input simply due to lack of time on my part :)

In short:
- I'm still looking after it in my spare time
- I don't plan to add new stuff unless either our needs force it or others make it happen
- its well tested.
- Will prob be one day totally redundant

Cheers, Stu

**Comment by johnraz** [comment]
Hey @stuartaccent,

Thank you so much for getting back to me so fast and thoroughly!
It really helped clarify the state of this repo.

Yes, sorry if I wasn't clear, the above repository is indeed the draft of the "official" solution from Wagtails, it's referenced here https://github.com/wagtail/wagtail/issues/4485#issuecomment-451188867.

I already asked there what the status of the project is on Wagtails's side.

Thanks again!
